Fixed prefix to /opt/<provider>/<package> form
See Filesystem Hierarchy Standard (http://www.debian.org/doc/packaging-manuals/fhs/fhs-2.3.html) for details
--- a/cmake/install_layout.cmake
+++ b/cmake/install_layout.cmake
@@ -74,7 +74,7 @@
   IF(INSTALL_LAYOUT MATCHES "RPM")
     SET(default_prefix "/usr")
   ELSEIF(INSTALL_LAYOUT MATCHES "DEB")
-    SET(default_prefix "/opt/mysql/server-${MYSQL_BASE_VERSION}")
+    SET(default_prefix "/opt/percona/percona-server-${MYSQL_BASE_VERSION}")
     # This is required to avoid "cpack -GDEB" default of prefix=/usr
     SET(CPACK_SET_DESTDIR ON)
   ELSEIF(INSTALL_LAYOUT MATCHES "SVR4")
